zulipterminal/ui.py

```diff
@@ -135,17 +136,18 @@ def main_window(self) -> Any:
         self.right_panel = self.right_column_view()
         if self.controller.autohide:
             body = [
-                (View.LEFT_WIDTH, self.left_panel),
+                (self.LEFT_WIDTH, self.left_panel),
                 ("weight", 10, self.center_panel),
                 (0, self.right_panel),
             ]
         else:
             body = [
-                (View.LEFT_WIDTH, self.left_panel),
+                (self.LEFT_WIDTH, self.left_panel),
                 ("weight", 10, self.center_panel),
-                (View.RIGHT_WIDTH, self.right_panel),
+                (self.RIGHT_WIDTH, self.right_panel),
             ]
         self.body = urwid.Columns(body, focus_column=0)
+        self.focus_col = self.body.focus_position
```

Comment:
This is basically to make the setting of focus_position shorter so that the code is easier to read as it is used many times.
Similarly for autohide as well.

Reply:
I understand it's useful for tracking the column, but it doesn't set the actually property. focus_col takes the value of body.focus_position, but setting the former again doesn't update the latter.

Reply:
But both of them point to the same property object right? So it would affect the object and they are not updating independent?

Reply:
I see it wasn't how I expected it to be. Is there a nice way to alias body.focus_position?

Reply:
Inside the object you could use self.body.focus_position; if setting from outside too then it may be clearer to have a method for it?